pkg/grpc/server.go

```diff
@@ -213,15 +213,18 @@ func (s *server) getMiddlewareOptions() []grpcGo.ServerOption {
 	if diagUtils.IsTracingEnabled(s.tracingSpec.SamplingRate) {
 		s.logger.Info("enabled gRPC tracing middleware")
 		intr = append(intr, diag.GRPCTraceUnaryServerInterceptor(s.config.AppID, s.tracingSpec))
-
-		if s.proxy != nil {
```

Comment:
Quick question why is this check removed? @yaron2

Reply:
This is from the time when the proxy was a preview feature and could have been nil. There is no way proxy can be nil now.